src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js

```diff
@@ -0,0 +1,188 @@
+/* global Promise */
+import { $, loc, createButton, View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
+import Logger from '../../../../util/Logger';
+import DeviceFingerprint from '../../../../util/DeviceFingerprint';
+import polling from '../shared/polling';
+import Util from '../../../../util/Util';
+
+const TotpBody = BaseForm.extend(Object.assign(
+  {
+    className: 'okta-verify-totp-challenge',
+
+    title () {
+      return loc('oie.okta_verify.totp.title', 'login');
+    },
+
+    save () {
+      return loc('mfa.challenge.verify', 'login');
+    },
+  },
+));
+
+const request = (opts) => {
+  const ajaxOptions = Object.assign({
+    method: 'GET',
+    contentType: 'application/json',
+  }, opts);
+  return $.ajax(ajaxOptions);
+};
+
+const UVDeviceChallengePoll = BaseForm.extend(Object.assign(
```

Comment:
We can move this to a separate file for readability

Reply:
Also before we go too far can you check with @jhe-okta  and @santoshmale-okta  why the remediation form name for UV challenge was challenge-authenticator and not 'device-challenge-uv-poll' or something. Coz UI is not sending anything here it just polls the backend similar to push.

Reply:
@jhe-okta can you please comment?

Reply:
Is `challenge-authenticator` added here https://github.com/okta/okta-signin-widget/pull/1372/files#diff-ca9bc08359fcf7d341af404edcca7e73R13?

Reply:
It is basically in the ViewFactory that it maps form names to views - https://github.com/okta/okta-signin-widget/pull/1374/files?file-filters%5B%5D=.js#diff-ba3ce0270635344ca7ec28ed51790688L132. Now with support for ov totp, the form name `challenge-authenticator` with authenticator type `app` will have different views depending on the method selected.

Reply:
No strong reason other than aligning with other remediations in this step. We have different remediations based on the platform like iOS/Windows etc. Some remediations ended up with just a polling url. Is the guideline that we don't name the form as `challenge-authenticator` unless the response from authenticator is coming back inline from SIW? If so, we can just rename it to `poll` wherever there is a polling url.